easybuild/easyconfigs/p/POV-Ray/POV-Ray-3.7.0.8-GCC-10.2.0.eb

```diff
@@ -0,0 +1,64 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'POV-Ray'
+version = '3.7.0.8'
+
+homepage = 'https://www.povray.org/'
+description = """The Persistence of Vision Raytracer, or POV-Ray, is a ray tracing program
+ which generates images from a text-based scene description, and is available for a variety
+ of computer platforms. POV-Ray is a high-quality, Free Software tool for creating stunning
+ three-dimensional graphics. The source code is available for those wanting to do their own ports."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/POV-Ray/povray/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = ['POV-Ray-3.7.0.7_dont-touch-home.patch']
+checksums = [
+    '53d11ebd2972fc452af168a00eb83aefb61387662c10784e81b63e44aa575de4',  # v3.7.0.8.tar.gz
+    '45103afca808e279dcdee80194c65e2a760c76d011d351392a46e817b0b655f7',  # POV-Ray-3.7.0.7_dont-touch-home.patch
+]
+
+dependencies = [
+    ('Boost', '1.74.0'),
+    ('zlib', '1.2.11'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.5'),
+    ('X11', '20201008'),
+    ('LibTIFF', '4.1.0'),
+    ('SDL2', '2.0.14'),
+]
+
+builddependencies = [
+    ('binutils', '2.35'),
```

Comment:
Doesn't need binutils at GCC/iccifort level

And please move builddeps ahead of deps.